### ISSUE TITLE
Pre-train masking

### ISSUE THREAD

**Issue by Pixelatory** [zhang-xuan1314/MTL-BERT#2]
With the implementation in dataset.py:

In Smiles_Bert_Dataset's numerical_smiles(), I see that there's an 80% chance of changing the encoding to the MASK token (as stated in the paper for this work), however it's set to 48 which is the <UNK> token correct (as by smiles_str2num)? Should this be 47 instead which is <MASK>?
